packages/ui/src/pages/SignIn/index.module.scss

```diff
@@ -27,6 +27,14 @@
   }
 }
 
+.placeHolderTop {
+  flex: 3;
+}
+
+.placeHolderBottom {
```

Comment:
nit: placeholder is one word. Don't need to capitalize the letter `H`

Reply:
updated